modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/UploadDocuments/UploadFiles.js

```diff
@@ -2,17 +2,27 @@ import ClayIcon from '@clayui/icon';
 import React, {useState} from 'react';
 
 import {InfoBadge} from '~/common/components/fragments/Badges/Info';
+import {RemoveDocument} from '../../../services/DocumentsAndMedia';
```

Comment:
Use our alias to simplify this '../../' -> '~/routes/selected-quote/services/DocumentsAndMedia'